### ISSUE TITLE
What changes in every update?

### ISSUE THREAD

**Issue by i-paya** [atilaromero/telegram-desktop-decrypt#8]
Hi, I tried to decrypt tdata file from latest version of telegram with this tool and it didn't work, and tried with older version and it worked and decrypted tdata file.
My question is what changes in every update in telegram desktop and what should I change in this tool to work?

**Comment by atilaromero** [comment]
I suspect the map file format has changed, but right now I don't have the time to figure out the details.

**Comment by engrobertofernandes** [comment]
hi, for me not even the older version works... the last version always says for each file "expected 3 streams, got 1"... maybe it is possible do decrypt just the 1 stream if it is a media file? 
If someday you have the chance or time to check that out or even update this library it would be helpful to a lot of people because i haven't found other updated libraries in internet that helps decrypt telegram tdata... thanks.

**Comment by blackdimund** [comment]
yeah im getting the `2020/09/11 18:25:49 expected 3 streams, got 1` error

**Comment by Higherupanddown** [comment]
What about now?
